clawpack/__init__.py

```diff
@@ -14,7 +14,11 @@
 
 __all__ = list(_subpackages.keys())
 
-_root = _os.path.dirname(__path__[0])
+CLAW_dir = _os.environ.get('CLAW')
+if CLAW_dir == None:
```

Comment:
Would a reasonable default directory be the current working directory?  Still would be good to raise a warning but most editable installs are probably being done from the eventual `$CLAW` directory.

Reply:
This code is executed when the package is imported, not when it is installed.